packages/ui/src/Bootstrap_PT.ts

```diff
@@ -196,5 +196,24 @@ export default {
     closeButton: () => ({
       class: ['ms-auto btn d-flex']
     })
+  },
+  badge: {
+    root: ({ props } : BadgePassThroughMethodOptions) => ({
+      class: [
+        'badge',
+        {
+          'text-bg-primary': props.severity === 'primary' || props.severity === null,
```

Comment:
is this a real class name?

Reply:
This is how they did it on the bootstrap 5 docs 

https://getbootstrap.com/docs/5.3/components/badge/#background-colors